### PR TITLE
fix: logging weights correctly in utils/weight_utils.py

### PR DESCRIPTION
**Pull request by grantfosterdev**
### Bug
#2361

### Description of the Change
Any instances where weights are being logged have been changed to `bt.logging.debug("weights", *weights)`.  The bug occurs when we attempt to log with `bt.logging.debug("weights", weights)`

### Alternate Designs
The only other alternative was deleting the logging lines - I chose to keep them, but simply log them correctly.

### Possible Drawbacks
None.

### Verification Process
I have run my validator on our testnet, starting from a completely fresh state file.  This means that I saw the validator correctly set weights when there were NO weights to set (i.e. all 1's), and when there WERE weights to set (i.e. miners have done some work and have been scored).

### Release Notes
Fixes logging bug where weights were being logged w/o an `*`

### CODE REVIEW

**Comment by thewhaleking** [comment]
Please update this to merge into staging.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+## What's Changed
+* remove commit from e2e tests by @roman-opentensor in https://github.com/opentensor/bittensor/pull/2340
+* add bittensor-cli as prod deps for sdk by @roman-opentensor in https://github.com/opentensor/bittensor/pull/2345
+* Fix the install command syntax by @rajkaramchedu in https://github.com/opentensor/bittensor/pull/2346
+* add config test by @roman-opentensor in https://github.com/opentensor/bittensor/pull/2347
+* Bumps version for 8.2.0 by @ibraheem-opentensor in https://github.com/opentensor/bittensor/pull/2348
+
+**Full Changelog**: https://github.com/opentensor/bittensor/compare/v8.1.1...v8.2.0
+
```
Comment (basfroman) [inline CHANGELOG.md]: those lines shouldn't be here
Otherwise gonna be LGTM

**Review by basfroman — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (grantfosterdev) [reply CHANGELOG.md]: this happened when I changed the base from master to staging... I never made those commits directly!

Comment (basfroman) [reply CHANGELOG.md]: Pls sync your branch with staging and remove all changes exclude logging related. then I gonna put LGTM
Thanks for this contributions.

**Review by basfroman — CHANGES_REQUESTED** [review]

**Review by basfroman — APPROVED** [review]

**Review by thewhaleking — APPROVED** [review]

**Review by basfroman — APPROVED** [review]

**Pull request merged**